File: rectangle.c
#include <stdio.h>

// Function to calculate the area of a rectangle
float calculateRectangleArea(float length, float width) {
	return length * width;
}

int main() {
	// Variables for lenght and width
	float length, width;
	
	// USer input
	printf("Enter the length of the rectangle: ");
	scanf("%f" ,&length);
	
	printf("Enter the width of the rectangle: ");
	scanf("%f" ,&width);
	
	// Calculate and display the area
	float area = calculateRectangleArea(length, width);
	printf("The area of the rectangle is: %.2f\n", area);
	
	return 0;
}

File: rectangle.py
# Function to calculate the area of a rectangle

def calculate_rectangle_area(length, width):
	return lenght  * width

# User input
length = float(input("Enter the length of the rectangle: "))
width = float(input("Enter the width of the rectangle: "))

# Calculate and display the area
area = calculate_rectangle_area(length, width)
print(f"The area of the rectangle is: {area:.2}")
